src/components/Books.js

```diff
@@ -5,6 +5,7 @@ import { addBook, removeBook, editBook } from '../redux/Books/booksSlice';
 function Books() {
   const dispatch = useDispatch();
   const books = useSelector((state) => state.books);
+  console.log(books);
```

Comment:
- [OPTIONAL] Please avoid the use of `console` statements while pushing your code for review. Having the `console` statement actually impacts the performance of the application. You are using only one so there is not serious issue, but having multiple can cause serious issues while developing big application. You can use it while building application but don't leave statement on staging of your changes.